e2e/ssr-basic.spec.ts

```diff
@@ -81,4 +81,35 @@ test.describe(`ssr-basic`, () => {
     await page.goto(`http://localhost:${port}/test`);
     await expect(page.getByTestId('resolved-promise')).toHaveText('test');
   });
+
+  test('test env', async ({ page }) => {
+    await page.goto(`http://localhost:${port}`);
+
+    const testEnvServer = page.getByTestId('test-env-server');
+    await expect(testEnvServer).toHaveText(
+      JSON.stringify(
+        {
+          WAKU_PUBLIC_TEST_META: 'ok',
+          WAKU_PRIVATE_TEST_META: '-',
```

Comment:
Isn't this unexpected? I wonder if it's broken at some point, or it's from the beginning.

Reply:
> In that sense, should `WKU_PRIVATE_...` be prohibited in the client even with `process.env`?

You mean preventing `process.env.WAKU_PRIVATE_...` on both SSR (available now) and Browser (unavailable now)? I think preventing SSR to access `process.env` is impossible, so the current behavior seems okay to me.